api/src/main/java/org/openmrs/module/appointments/service/impl/AppointmentsServiceImpl.java

```diff
@@ -373,12 +404,12 @@ public Appointment reschedule(String originalAppointmentUuid, Appointment newApp
 
         try {
             //cancel the previous appointment
-            changeStatus(prevAppointment, AppointmentStatus.Cancelled.toString(), new Date());
+            changeStatus(prevAppointment, AppointmentStatus.Rescheduled.toString(), new Date());
```

Comment:
Defer to others on this, or whether Cancelled -> Scheduled is the desired approach, and determining whether an appointment is "rescheduled" is done by inspecting the audit history.